src/integration/tests/firstTest.test.tsx

```diff
@@ -1,5 +1,4 @@
-//import React from "react";
-import { render, screen, within, waitFor} from "@testing-library/react";
+import { render, screen, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import App from "../../App";
```

Comment:
Please add an empty line between libs imports and local imports

```suggestion
import userEvent from "@testing-library/user-event";

import App from "../../App";
```